src/charm.py

```diff
@@ -198,6 +198,13 @@ def service_extra_args(self, service_name, config_key) -> str:
         args = self.cis_benchmark.craft_extra_args(service_name, extra_args)
         return " ".join(f"{k}={v}" for k, v in args.items())
 
+    def allow_privileged(self) -> bool:
+        val = self.model.config["allow-privileged"].lower()
+        if val not in ["true", "false", "auto"]:
+            status.add(ops.BlockedStatus("Invalid config for allow-privileged"))
+            return False
+        return val == "true"
```

Comment:
* Only `value == true` allows it
* invalid config sets unit to a blocked state, but the cluster can still go active with `allow-privileged=False`